R/vars_manager.R

```diff
@@ -244,18 +305,20 @@ format_vars <- function(envir, abbrev_len=DEFAULT_ABBREV_LEN) {
 #' @param envir An environment 
 #' @param name Name of a variable in the environment
 #' @param abbrev_len The length of elements at which vars should be abbreviated. Pass NULL to prevent abbreviation.
+#' @param sort_by A vector containing columns to sort (primary first)
+#' @param ascending A vector of booleans or a single boolean, depending on the data type
```

Comment:
how do I get these to appear in the documentation? It looked like we use `roxygen2` for that?